### PR TITLE
MM-50417 Fixed centre channel being visible before channels are loaded with GraphQL

### PR DESCRIPTION
**Pull request by hmhealey**
I changed around how the loading screen that blocks the `CenterChannel` works in https://github.com/mattermost/mattermost-webapp/pull/12058, and for whatever reason, I made it skip the loading screen that's shown while initially loading if GraphQL is enabled. I can't remember exactly why I would've done that, but I must've thought that the user's channels were loaded by this point.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-50417

#### Related Pull Requests
https://github.com/mattermost/mattermost-webapp/pull/12058

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "mattermost-webapp",
```
Comment (larkox) [inline package-lock.json:8]: Is this change intended? Seems like the kind of change that different versions of npm would add/remove between PRs.

**Review by larkox — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (hmhealey) [reply package-lock.json:8]: Not really for this, but I didn't want to bother submitting a separate PR for that.

It keeps getting added and removed periodically. I think it's either caused by someone's NPM  being the wrong version or a difference between `npm install` and `npm add` again.

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by stevemudie — APPROVED** [review]
This is all set @hmhealey!
Tested and worked as expected.
